expyfun/_experiment_controller.py

```diff
@@ -843,7 +852,8 @@ def flip(self, when=None):
         # this waits until everything is called, including last draw
         gl.glClear(gl.GL_COLOR_BUFFER_BIT)
         gl.glBegin(gl.GL_POINTS)
-        gl.glColor4f(0, 0, 0, 0)
+        if not self._enable_video:
+            gl.glColor4f(0, 0, 0, 0)
```

Comment:
I decided not to change this right now, in the interest of getting on with my experiment.  Will open a TODO issue for testing video / drawing / audio synchronization, and implement separation this at that point if needed.  Check the new example though: drawing text on top of video seems to work fine, so I'm not convinced that the separation is necessary.